modules/govuk_elasticsearch/templates/es-restore-s3.erb

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+# Redirect stdout and stderr to syslog
+exec 1> >(/usr/bin/logger -s -t $(basename $0)) 2>&1
+
+ES_REPO=<%= @es_repo %>
+ES_INDICES=<%= @json_es_indices %>
+ES_SNAPSHOT=snapshot_$(date +%d_%m_%y)
+
+# PARAMETERS FOR ELASTICSEARCH SNAPSHOT RESTORE
+RESTORE_DATA=$(cat <<EOD
+{
+  "indices": "${ES_INDICES}",
+  "ignore_unavailable": true,
+  "include_global_state": false,
+  "rename_pattern": "index_(.+)",
+  "rename_replacement": "restored_index_$1"
+}
+EOD
+)
+
+# List the snapshots and grab the last one
+LATEST_SNAPSHOT=$(/usr/bin/curl --connect-timeout 10 -sS -XGET "127.0.0.1:9200/_snapshot/${ES_REPO}/_all?pretty" | /usr/bin/jq --raw-output ".snapshots[].snapshot" |tail -1)
+
+curl -XPOST "localhost:9200/_snapshot/${ES_REPO}/${LATEST_SNAPSHOT}/_restore" -d $"${RESTORE_DATA}"
```

Comment:
I think maybe I misunderstand the way this works. I thought the S3 data was contained in the repo, and so when restoring the snapshot it gathers this data when you specify which repo you're restoring from. Where do I need to further specify the data source?